megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -1202,7 +1202,7 @@ private String checkGuess(final Entity shooter,
                 String shootingCheck;
 
                 // Energy / ammo-independent weapons
-                if (AmmoType.T_NA == wtype.getAmmoType()) {
+                if (Arrays.asList(AmmoType.T_NA, AmmoType.T_INFANTRY).contains(wtype.getAmmoType())) {
```

Comment:
Should we split this logic into a function for a little more readability?

Reply:
Couldn't hurt, and we can make future updates less painful.  One sec...